usr/share/rear/conf/GNU/Linux.conf

```diff
@@ -164,6 +164,18 @@ sysctl
 blockdev
 lsblk
 clear
+
+# Older releases of os-prober (1.74 and below) use dmsetup as a fallback
+# solution for mounting when grub-mount is missing.
+#
+# However, dmsetup was included in the rescue image if and only if LVM,
+# multipath or encryption were detected.  Thus, BIOS machines that do
+# not use these but still have dmsetup present, would block indefinitely
+# on the "Installing GRUB2 boot loader..." step.
```

Comment:
maybe you should explain that it is because dmsetup is in the recovered system (chroot) but not in rescue system (outside chroot))? Otherwise it is far from obvious.

Reply:
Good catch! I've accidentally forgot to add the last paragraph from the PR description.